src/NServiceBus.Testing/TestingContext.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using NServiceBus.Testing.ExpectedInvocations;
+
+    class TestingContext : TestableMessageHandlerContext
```

Comment:
> What would be the harm of exposing TimeoutMessage also on the TestableMessageHandlerContext?

TimeoutMessage only captures timeouts sent by `Saga.RequestTimeout`, I wasn't sure whether we really want to include saga only stuff in the general context. Also the messages by `RequestTimeout` are contained in SentMessages and TimeoutMessages, TimeoutMessages actually need to be passed to next context by the testing framework to provide the assertions which definitely isn't something users need to or should do.